main.go

```diff
@@ -1,7 +1,44 @@
-package jotapi
+package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/labstack/echo/v4"
+)
 
 func main() {
-	fmt.Println("Hello from the jot api!")
-}
\ No newline at end of file
+	e := echo.New()
+
+	CreateHandler(e)
+
+	serverPort := os.Getenv("SERVER_PORT")
+	if serverPort == "" {
+		serverPort = "8080"
+	}
+
+	fmt.Println("Listening on port: ", serverPort)
+
+	err := e.Start(":" + serverPort)
+	if err != nil {
+		return
+	}
+}
+
+func CreateHandler(e *echo.Echo) {
```

Comment:
squibble: this comment is extremely nitpicky, but "CreateHandler" is not quite accurate. Most go devs use the term "handler" to refer to a function/object that handles a single endpoint. Plus most "create" functions typically return something that was created. So maybe rename it to something like "addRouteHandlers" 

Also, there's no reason to capitalize the function since that would mean it's exported and 1) you aren't importing it anywhere and 2) you can't import anything from the `main` package anyway